ember/app/ui/components/membership-role-form/template.hbs

```diff
@@ -0,0 +1,36 @@
+<div
+  {{did-insert this.onUpdate}}
```

Comment:
Wouldn't this be more appropriate in the constructor since it looks like setup work?

Reply:
Would the constructor rerun if `@role` changes?

Reply:
I thought so but i will check quickly.

Reply:
You're right the constructor has no args update.